sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1999,6 +1999,34 @@ class Dataset[T] private[sql](
   }
 
   /**
+   * Returns a new [[DataFrame]] which contains the aggregated result of applying
+   * [[func]] R function to each group
+   *
+   * @group func
+   * @since 2.0.0
+   */
+  def mapGroupPartitionsInR[K: Encoder](
+    gfunc: T => K,
+    func: Array[Byte],
+    packageNames: Array[Byte],
+    broadcastVars: Array[Object],
+    outputSchema: StructType): DataFrame = {
```

Comment:
The style issue. https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Indentation